### ISSUE TITLE
Mercury Trade hotkeys not working

### ISSUE THREAD

**Issue by GrumpyWarrior** [Exslims/MercuryTrade#328]
Hello! First off, thank You for this wonderful program. Till now it worked like a charm however as of lately (today) the "wisper helper" fuctionality suddenly stopped working. I wen't to Mercury options to perhabs change hotkey from F2 to something else, however it does not allow me. Whenever I try to change any key nothing happends as if program is completely "deaf". Even for "Incomming/Outgoing notification". Assigning hotkeys simply do not work and neither does functionality of pre-existing ones.
Everything else works completely fine. Notification windows are poping up and I can click buttons on it.

I tried clean install by deleting files from %appdata%local and downloading fresh file. I also tried to run Mercury via exe and also as administrator. Nothing seems to work. I can not assign any keybinds or use them.

**Comment by GrumpyWarrior** [comment]
Ok. Very wierd. I have restarted the computer and it seems to work now. Before I restarted my computer however I noticed strange thing. I could not rename any folder or file. I could not change properties or anything. Even in folders that required no administrator.
Now everything works. I have no idea what that was :(

Sorry for wasting your time!
Wish You a nice upcomming Christmas and a new year!
